python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/defs_additional_paths.schema.yml

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+type: dict
+$defs:
+  additional_paths:
+    type: dict
+    keys:
+      receive:
+        type: bool
+      send:
+        type: str
+        description: |-
+          Select an option to send multiple paths for same prefix through bgp updates.
+          any: Send any eligible path.
+          backup: Best path and installed backup path.
+          ecmp: All paths in best path ECMP group.
+          limit: Limit to n eligible paths.
+        valid_values: [ 'any', 'backup', 'ecmp', 'limit', 'null' ]
```

Comment:
I have added 'null' value to generate `no neighbor <neighbor_name> additional path send any` command. Does this look ok?

Reply:
As discussed, change it to `disabled`

Reply:
Done